### PR TITLE
WinUI 890011 - Updated Scheduler tooltip and cell padding content

### PR DESCRIPTION
**Pull request by TamilarasanGunasekaran**
I have added scheduler tooltip support and cell padding support content.

### CODE REVIEW

File: winui/Scheduler/Tooltip.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    </syncfusion:SfScheduler.ToolTipTemplate>
+</syncfusion:SfScheduler>
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainWindow.xaml.cs" hl_lines="2" %}
+  SfScheduler Schedule = new SfScheduler();      
+  Schedule.EnableToolTip = true;
+  this.Content = Schedule;
+{% endhighlight %}
+{% endtabs %}
```
Comment (Karthickmani97) [inline winui/Scheduler/Tooltip.md:70]: Mention in note - Tooltip template only be applicable when `EnabledToolTip` property is set to true.

File: winui/Scheduler/Tooltip.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+      </syncfusion:SfScheduler>
+{% endhighlight %}
+{% highlight c# %}
+  SfScheduler Schedule = new SfScheduler();      
+  Schedule.EnableToolTip = true;
+  this.Content = Schedule;
+{% endhighlight %}
+{% endtabs %}
+
+## Customize tooltip appearance using DataTemplate
```
Comment (Karthickmani97) [inline winui/Scheduler/Tooltip.md:31]: Add this in appointment md file

**Review by MuniappanSubramanian — APPROVED** [review]

**Review by Karthickmani97 — APPROVED** [review]
This is fine.

**Review by Jeyasri-Murugan — APPROVED** [review]

File: winui/Scheduler/Appointments.md [hunk 3]
```diff
@@ -756,6 +756,64 @@ N>
 
 N> [View sample in GitHub](https://github.com/SyncfusionExamples/WinUI-Scheduler-Examples/tree/main/RecursiveExceptionAppointment/BusinessObject)
 
+## Appointment tooltip
+
+An interactive tooltip provides additional details about the appointments on hovering the mouse over them.
+
+### Enable tooltip for appointments
+
+To enable tooltip for the scheduler appointments, use the `EnableTooltip` property of `SfScheduler.` By default, `EnableTooltip` is set to `false.` To provide users with additional information or context about appointments, simply set this property to true.
```
Comment (Jeyasri-Murugan) [inline winui/Scheduler/Appointments.md:765]: Follow to merge API link, once published

**Pull request merged**